README.md

```diff
@@ -95,7 +97,7 @@ Requests to operations which expect a request body will be validated, for exampl
 http --json POST http://127.0.0.1:4010/pets name=Stowford
 ```
 
-This will generate an error:
+This will generate an error, conforming the [application/problem+json][rfc7807] specification:
 
 ```
 Here is the original validation result instead: [{"path":["body"],"name":"required","summary":"should have required property 'photoUrls'","message":"should have required property 'photoUrls'","severity":"error"}]
```

Comment:
This could be an example of the awesome new problems api errors now.

Reply:
@philsturgeon You've got it in b1faf17